### PR TITLE
CB-925: Fix some logging levels, remove unused node modules, fix memory leak

### PR DESCRIPTION
**Pull request by joshuadiablito**

### CODE REVIEW

**Comment by joshuadiablito** [comment]
@aminmc this commit: https://github.com/UKHomeOffice/cop-private-workflow-tasklist/pull/250/commits/c3800f8097a5d5eb29eff286f83fa0e212154fc3 I believe fixes a potential memory leak where the subscriptions aren't cleared up - can you take a look for me please?

File: app/core/FileService.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ class FileService {
         const fd = new FormData();
 
         const data = {
-            file: file,
+            file,
             name: fileName,
-            dir: dir
+            dir
```
Comment (p3dr0migue1) [inline app/core/FileService.js]: :+1:

File: app/core/FileService.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ class FileService {
     }
 
     async getToken() {
-        let token = this.keycloak.token;
+        let {token} = this.keycloak;
```
Comment (p3dr0migue1) [inline app/core/FileService.js]: nice and clean

File: app/core/components/SubmissionBanner.jsx [hunk 3]
```diff
@@ -38,27 +38,31 @@ class SubmissionBanner extends React.Component {
...
            </div>
-         </div>
-       } else {
-         return <div className="container" id="successfulSubmission" style={{paddingTop: '5px'}}>
-           <div className="govuk-panel govuk-panel--confirmation">
-             <div className="govuk-panel__body govuk-!-font-size-24 govuk-!-font-weight-bold">
-               {message}
+)
+       } 
+         return (
```
Comment (p3dr0migue1) [inline app/core/components/SubmissionBanner.jsx]: :+1:

File: app/core/shift/shift.reducer.js [hunk 4]
```diff
@@ -41,7 +41,7 @@ function shiftReducer(state = shiftInitialState, action) {
     case actions.SUBMIT_VALIDATION:
       return state.set('submittingActiveShift', true).set('failedToCreateShift', false);
     case actions.CREATE_ACTIVE_SHIFT_SUCCESS:
-      const shift = JSON.parse(action.payload.entity.variables['shiftInfo'].value);
+      const shift = JSON.parse(action.payload.entity.variables.shiftInfo.value);
```
Comment (p3dr0migue1) [inline app/core/shift/shift.reducer.js]: nice catch

File: package.json [hunk 5]
```diff
@@ -58,9 +57,6 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "braces": "^3.0.2",
```
Comment (p3dr0migue1) [inline package.json]: nice clean up

**Pull request merged**